zds/utils/management/commands/load_fixtures.py

```diff
@@ -35,13 +35,13 @@
 
 def load_member(cli, size, fake, root, *_):
     """
-    Load members
+    Load members, including a portion as spam profiles.
     """
-    nb_users = size * 10
-    cli.stdout.write(f"Nombres de membres à créer : {nb_users}")
+    nb_users = size * 50
+    spam_ratio = 0.2  # 20% of profiles will be spam
+    cli.stdout.write(f"Nombres de membres à créer : {nb_users} (including {int(nb_users * spam_ratio)} spammers)")
```

Comment:
```suggestion
    cli.stdout.write(f"Nombres de membres à créer : {nb_users} (dont {int(nb_users * spam_ratio)} spammeurs)")
```
(pour rester cohérent)

Reply:
Fixed!